src/main/java/net/dries007/tfc/CommonEventHandler.java

```diff
@@ -192,12 +266,12 @@ public static void onPlayerLoggedIn(PlayerEvent.PlayerLoggedInEvent event)
             // World Data (Calendar) Sync Handler
             TerraFirmaCraft.getNetwork().sendTo(new PacketCalendarUpdate(), player);
 
-            // Player nutrients
-            IPlayerNutrients cap = player.getCapability(CapabilityFood.CAPABILITY_PLAYER_NUTRIENTS, null);
+            // Player data
+            IPlayerData cap = player.getCapability(CapabilityPlayer.CAPABILITY_PLAYER_DATA, null);
             if (cap != null)
             {
-                cap.updateNutrientsFastForward();
```

Comment:
Why was this call removed? It was specifically there to avoid players who had been logged out of a server for a while from getting a massive decay drop

Reply:
it wasn't, I refactored the method to onUpdate()